src/test/java/uk/gov/justice/hmpps/datacompliance/web/controllers/RetentionControllerIntegrationTest.java

```diff
@@ -34,6 +34,46 @@
     @Autowired
     private JwtAuthenticationHelper jwtAuthenticationHelper;
 
+    @Test
+    @Sql("retention_reason_code.sql")
```

Comment:
not seen this before, what does it do?

Reply:
@andymarke it allows the db to be seeded with test data pertinent to only the test in question - this references a file in the test resources (matching the test's package).

https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/test/context/jdbc/Sql.html